### PR TITLE
Add Client skeleton

### PR DESCRIPTION
**Pull request by heartjigi**
1. add README.md 
2. solve problem of ".babelrc crash"

@ethkim @nlpenguin @djflexible @ifndefdeadmau5 @ace0625 @MechanicKim

### CODE REVIEW

File: src/client/index.android.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+ */
+
+import React, { Component } from 'react';
+import {
+  AppRegistry
+} from 'react-native';
+
+import App from './javascripts/App';
+
+//regist to AppRegistry for route
```
Comment (ethkim) [inline src/client/index.android.js]: Register. 
1 empty space after //

// Register route to AppRegistry

File: src/client/index.ios.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+ */
+
+import React, { Component } from 'react';
+import {
+  AppRegistry
+} from 'react-native';
+
+import App from './javascripts/App';
+
+//regist to AppRegistry for route
```
Comment (ethkim) [inline src/client/index.ios.js]: ditto

File: src/client/ios/YodaTests/YodaTests.m [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+    if ([self findSubviewInView:subview matching:test]) {
+      return YES;
+    }
+  }
+  return NO;
+}
+
+- (void)testRendersWelcomeScreen
+{
+  UIViewController *vc = [[[[UIApplication sharedApplication] delegate] window] rootViewController];
```
Comment (ethkim) [inline src/client/ios/YodaTests/YodaTests.m]: This code (and below) is really hard to read. Was this generated?

Comment (heartjigi) [inline src/client/ios/YodaTests/YodaTests.m]: yes it's generated by react-native.
you can pass this time.

**Comment by ethkim** [comment]
Thanks Dabin!

@ifndefdeadmau5 Could you please take a look at this? (I'll leave the final decision on the PR to you).

I've made several comments about the style in previous PRs (they're closed now but should be searchable again). Please respond to those in addition to these. Other than that, the overall structure looks good.

Once this is checked in, we'll also have to put in the framework for unittests. Could you add a Trello item for adding tests for the client, and assign it to the frontend team? Thanks!

**Comment by heartjigi** [comment]
@ethkim 
Hi DongHyun
I changed code you mentioned at closed PR
and I delete *.imi ( you want to break line )  because it's not nessary
Thanks!

**Comment by ethkim** [comment]
Thanks Dabin!

LGTM though Travis seems to be broken. This is odd since we don't have any travis tests on the client side. Am I missing something?

**Comment by ethkim** [comment]
I think it's missing the setting for react-native inside the client subdirectory. Could you take a look?

Error: Couldn't find preset "react-native" relative to directory "/home/travis/build/sv-bootcamp/yoda/src/client"
    at /home/travis/build/sv-bootcamp/yoda/node_modules/babel-core/lib/transformation/file/options/option-manager.js:281:17
    at Array.map (native)
    at OptionManager.resolvePresets (/home/travis/build/sv-bootcamp/yoda/node_modules/babel-core/lib/transformation/file/options/option-manager.js:265:20)
    at OptionManager.mergePresets (/home/travis/build/sv-bootcamp/yoda/node_modules/babel-core/lib/transformation/file/options/option-manager.js:254:10)
    at OptionManager.mergeOptions (/home/travis/build/sv-bootcamp/yoda/node_modules/babel-core/lib/transformation/file/options/option-manager.js:239:14)
    at OptionManager.init (/home/travis/build/sv-bootcamp/yoda/node_modules/babel-core/lib/transformation/file/options/option-manager.js:338:12)
    at File.initOptions (/home/travis/build/sv-bootcamp/yoda/node_modules/babel-core/lib/transformation/file/index.js:216:65)
    at new File (/home/travis/build/sv-bootcamp/yoda/node_modules/babel-core/lib/transformation/file/index.js:137:24)
    at Pipeline.transform (/home/travis/build/sv-bootcamp/yoda/node_modules/babel-core/lib/transformation/pipeline.js:46:16)
    at DestroyableTransform._transform (/home/travis/build/sv-bootcamp/yoda/node_modules/gulp-babel/index.js:37:20)

**Comment by ethkim** [comment]
LGTM

**Comment by KaylaKwon** [comment]
Congrats!

**Pull request merged**